src/send/usePrepareSendTransactions.ts

```diff
@@ -32,22 +33,26 @@ export async function _prepareSendTransactionsCallback({
   if (amount.isLessThanOrEqualTo(0)) {
     return
   }
+  const baseTransactionParams = {
+    // not including sendToken yet because of typing. need to check whether token has address field or not first, required for erc-20 transfers
```

Comment:
this is because erc-20 transfer methods require TokenBalanceWithAddress right? Looks like it's deprecated, should it no longer be deprecated?

Reply:
strangely, the same file exports two interfaces with the same name, only one of which is deprecated. I'm surprised that's possible. Cleaning it up...